backend/app/api/routes/question.py

```
@@ -765,119 +765,113 @@
     )
 
 
-@router.post("/{question_id}/locations", response_model=QuestionLocationPublic)
-def add_question_location(
+@router.post("/{question_id}/locations", response_model=list[QuestionLocationPublic])
+def add_question_locations(
```

Pylint (3.3.7)

[refactor] Too many branches (18/12)

(R0912)